generators/app/index.js

```diff
@@ -322,13 +327,16 @@ module.exports = generators.Base.extend({
           this.error404 = answers.error404 ? answers.error404 :  this.options.error404 || '404.js';
           this.error5xx = answers.error5xx ? answers.error5xx :  this.options.error5xx || '5xx.js';
           this.error503 = answers.error503 ? answers.error503 :  this.options.error503 || '503.js';
-          this.staticsRoot = answers.staticsRoot ? answers.staticsRoot :  this.options.staticsRoot || 'statics';
+          this.staticsRoot = answers.staticsRoot ? answers.staticsRoot :  this.options.staticsRoot || 'statics/';
           this.htmlMinify = this.options.htmlMinify || '{"override": true, "exception_url": false, "htmlMinifier": {"html5": true}}';
           this.cssPath = answers.cssPath ? answers.cssPath :  this.options.cssPath || 'css';
           this.cssCompiler = answers.cssCompiler ? answers.cssCompiler :  this.options.cssCompiler || '{"nodeModule": "roosevelt-less", "params": {"compress": true}}';
           this.cssCompilerWhitelist = answers.cssCompilerWhitelist ? answers.cssCompilerWhitelist :  this.options.cssCompilerWhitelist || 'null';
           this.cssCompiledOutput = answers.cssCompiledOutput ? answers.cssCompiledOutput :  this.options.cssCompiledOutput || '.build/css';
           this.jsPath = answers.jsPath ? answers.jsPath :  this.options.jsPath || 'js';
+          this.bundledJsPath = answers.bundledJsPath ? answers.bundledJsPath :  this.options.bundledJsPath || '.bundled';
+          this.exposeBundles = answers.exposeBundles ? answers.exposeBundles :  this.options.exposeBundles || 'true';
+          this.browserifyBundles = answers.browserifyBundles ? answers.browserifyBundles :  this.options.browserifyBundles || '[]';
```

Comment:
The htmlMinify param is being installed wrong. Coming up as `"htmlMinify": "htmlMinify"` in a fresh sample app. My guess based on line 331 is that `this.options.htmlMinify` is set to that string and preventing the default from being set.